frb/dm/prob_dmz.py

```diff
@@ -176,6 +176,32 @@ def grab_repo_grid():
     return sdict
 
 
+
+def grab_chime_repo_grid():
+    """
+    Grab the CHIME grid from the Repository
+
+    Returns:
+        dict: Numpy dict from the npz save file
+    """
+
+    # File
+    CHIME_PDM_z_grid_file = os.path.join(
+        resource_filename('frb', 'data'), 'DM',
+        'CHIME_pzdm.npz')
+
+    # assert the file exists
+    if not os.path.isfile(CHIME_PDM_z_grid_file):
```

Comment:
it looks like you have added the file into the Repo (which I guess is fine).

so, can this happen?

Reply:
solved